xrdp/xrdp_mm.c

```diff
@@ -3610,14 +3621,11 @@ server_send_to_channel(struct xrdp_mod *mod, int channel_id,
 
     if (wm->mm->usechansrv)
     {
-        /*
-         * Xvnc backend reaches here
-         * should not return 1 as this case is not an error
-         */
-        return 0;
+        /* Modules should not be calling this if chansrv is running -
+         * they can use server_chansrv_in_use() to avoid doing this */
+        return 1;
```

Comment:
I think we should add a `LOG_DEVEL` warning message to notify devs of the programming error.

Reply:
Agreed.